mta/application.properties

```diff
@@ -1,8 +1,7 @@
-mta.endpoint = mta-ui.openshift-mta.svc.cluster.local
-mta.port = 8080
-mta.protocol = http
+mta.endpoint = ${MTA_ENDPOINT:mta-ui.openshift-mta.svc.cluster.local:8080}
+mta.protocol = ${MTA_PROTOCOL:http}
 mta_url = ${mta.protocol}://${mta.endpoint}
-quarkus.rest-client.mta_json.url = ${mta.protocol}://${mta.endpoint}:${mta.port}/hub
+quarkus.rest-client.mta_json.url = ${mta.protocol}://${mta.endpoint}/hub
```

Comment:
why is the `mta_url` needed?
can't it be used instead of `${mta.protocol}://${mta.endpoint}` ? 
meaning `${mta_url}/hub` ?

Reply:
`mta_url` is used directly in the notification 

I would want to even simply it further by having only the property `mta.url` (which will be  like ${mta.protocol}://${mta.endpoint})

Reply:
I did that ^^^^

Reply:
sorry, I meant to having something like:
```
quarkus.rest-client.mta_json.url = ${mta_url}/hub
```